### PR TITLE
Update the action bar on manage contents page

### LINKED ISSUE #12367: Improve the look of the action bar in all the views
### Describe the bug
When using a mobile phone to manage contents (or any view with action bar), the search input on the top shows up very small.

image

Also, on the filter button on the left, can we remove the round corner between the button and the input?

image

### PR DESCRIPTION
**Pull request by MikeAlhayek**
Fix #12367

## Web view

Here is a before screenshot 
image

Here is an after screenshot
image

## Mobile view
Here is a before screenshot
image

Here is an after schreenshot
image

### CODE REVIEW

**Comment by hishamco** [comment]
Can we increase the search bar little to make the spacing consistent

**Comment by MikeAlhayek** [comment]
@hishamco I made some more changes to the second action bar along with the action bar spacing. check out the updated screenshots

**Comment by Skrypt** [comment]
Can we see what the searchbar looks like now when width = 100%?

**Comment by Skrypt** [comment]
Also, need to see if these UI fixes need to be applied everywhere else.

**Comment by MikeAlhayek** [comment]
@Skrypt the PR #12368 has the before and after screenshots.

image

image

**Comment by Skrypt** [comment]
can you try and add a w-md-50 or w-lg-50 class there.

File: src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsAdminListSearch.cshtml
Comment (Skrypt) [inline src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsAdminListSearch.cshtml:6]: here try w-md-50 or w-lg-50 instead.

**Comment by MikeAlhayek** [comment]
I personally think it looks better with 100% width.

**Comment by Skrypt** [comment]
I can't argue with taste. If you have a 4k screen it will make a long search input bar. That's why it was added.

**Comment by Skrypt** [comment]
Here as an example Github does not use full width:

image

**Comment by Skrypt** [comment]
Unless you would need to type long search strings that are equivalent to the full width size of a 4k screen I don't see the added value either.

**Review by Skrypt — APPROVED** [review]

**Comment by hishamco** [comment]
I'm not sure if I'm the only one see extra space before the new dropdown

**Comment by Skrypt** [comment]
Add a screenshot. I can't see it either! 😄

**Comment by hishamco** [comment]
I think there's two extra pixels before the new dropdown comparing with the spacing after, needs BIG 👀

**Comment by Skrypt** [comment]
I don't see any extra pixels in his screenshots.

**Comment by hishamco** [comment]
@MikeAlhayek could you please inspect the spacing before and after the new dropdown? It looks good but that's why designer is better than devs because they taking care about each pixel :)

**Comment by MikeAlhayek** [comment]
@hishamco I did not add a drop down menu. Can you show us where are you pointing to? I would say share a screen shot with the develops so that they can see through a designer eyes :)

**Comment by Skrypt** [comment]
The filter button?

**Comment by hishamco** [comment]
I just measure the spaces before and after the new dropdown seems the quality here as @Skrypt mentioned, both are 16 px, so seems I need to close my eyes ;)

**Comment by hishamco** [comment]
@MikeAlhayek could you please check tenants page? I remembered I saw similar issue

**Comment by MikeAlhayek** [comment]
@hishamco yea same problem exists in multiple pages. I want to focus on Contents module for now, once all agree on the change and is merged, I'll apply the same on the rest like OC.Tenant, OC.Features and possibly others.

**Comment by hishamco** [comment]
We already agreed ;) I might help you on this

Thanks

**Review by hishamco — APPROVED** [review]

File: src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsAdminListSearch.cshtml
Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsAdminListSearch.cshtml:6]: @Skrypt are you fine to merge this without this suggestion?

**Comment by hishamco** [comment]
@MikeAlhayek I just waiting for @Skrypt to reply on my above comment meanwhile could you please file an issue list all the places that we need to modify the action bar, this will make it easy for tracking and I can help you on this especially if it has a long list

File: src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsAdminListSearch.cshtml
Comment (MikeAlhayek) [reply src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsAdminListSearch.cshtml:6]: @Skrypt if you must have a max width set, we could add `w-xxl-75` just for super large screens.

**Comment by MikeAlhayek** [comment]
Combined into #12421

**Comment by hishamco** [comment]
I'm not sure why closed the PRs, small manageable PRs always good

**Pull request closed without merging**
